virtwho/virt/libvirtd/libvirtd.py

```diff
@@ -115,13 +115,14 @@ def isHypervisor(self):
 
     def _get_url(self):
         if self.config.get('server', None):
+            # FIXME: Following code can be removed, when ConfigSection will be used
```

Comment:
We should remove this now as the LibvirtdConfigSection should be in use after this PR is merged.